server/src/ConfigManager.cc

```diff
@@ -78,6 +79,33 @@ static void showVersion(void)
 	       VERSION, __DATE__, __TIME__);
 }
 
+static gboolean parseFaceRestWorkers(
+  const gchar *option_name, const gchar *value,
+  gpointer data, GError **error)
+{
+	GQuark quark =
+	  g_quark_from_static_string("config-manager-quark");
+	CommandLineOptions *obj =
+	  static_cast<CommandLineOptions *>(data);
+	if (!value) {
+		g_set_error(error, quark, CONF_MGR_ERROR_NULL,
+		            "value is NULL.");
+		return FALSE;
+	}
+
+	int n = atoi(value);
```

Comment:
The variable name 'n' is too short to easy recognize what it is.
How about using numWorks.